velox/functions/prestosql/StringFunctions.h

```diff
@@ -120,6 +120,18 @@ struct Sha512Function {
   }
 };
 
+/// hmac_sha1(varbinary) -> varbinary
+template <typename T>
+struct HmacSha1Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TOuput, typename TInput>
+  FOLLY_ALWAYS_INLINE void
+  call(TOuput& result, const TInput& data, const TInput& key) {
+    return stringImpl::HmacSha1(result, key, data);
```

Comment:
call return void?
```
s/return stringImpl::HmacSha1(result, key, data);/stringImpl::HmacSha1(result, key, data);/
```

Reply:
I see https://github.com/facebookincubator/velox/pull/3406 has changed the return type of HmacSha256Function to `void`, so I Modified it in advance,